tests/metrics/README.md

```diff
@@ -4,25 +4,19 @@
 Some tests require you to enable the debug mode of cc-oci-runtime. To enable it, please take a look at:
 https://github.com/01org/cc-oci-runtime#18running-under-docker
 
-To run the metrics, just run:
+
+### Run all metrics tests
+To run the metrics, please use Makefile rule `metrics-tests` on the top level of this repository:
 
 ```bash
-$ cd tests/metrics
-$ ./run_docker_metrics
+$ sudo make metrics-tests
```

Comment:
Just a note - this does not quite work on my 'ubuntu under ciao-down' instance as 'go' is not available to root, and this make line tries to build the whole source base (from a clean checkout), including the proxy which is in `go`. Just a niggle/annoyance.